Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/AdminDistTransforms.java

```diff
@@ -100,8 +100,7 @@ protected LogEventRequestType getLogEventRequestType(EDXLDistribution body, Asse
 
         result.setAuditMessage(auditMsg);
         result.setDirection(direction);
-        result.setInterface(_interface);
-
+        result.setRemoteHCID(HomeCommunityMap.formatHomeCommunityId(communityId));
```

Comment:
Edit: For AD we are still using interface to find out the remoteHCID. As discussed offline, we will keep the getAdminDistributionMessageCommunityID() method as is.